cli/cmd/plugin/diagnostics/e2e-test/e2e_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
```

Comment:
Should add tag here so the end to end tests are not run as part of running unit tests:

https://github.com/vmware-tanzu/community-edition/blob/main/cli/cmd/plugin/unmanaged-cluster/test/e2e/e2e_suite_test.go#L1